README.md

```diff
@@ -0,0 +1,1075 @@
+# node-diacritics-transliterator
+
+> Diacritic transliteration tools using diacritics.io API
```

Comment:
I think that this is the most important sentence in the whole documentation and we need to elaborate a bit more what a user can do with this module. I think he won't read through the documentation to find out what it's about, so we should clarify this directly at the beginning to win him as a user. What do you think?